### DISCUSSION TITLE
Distribution of stats

### DISCUSSION THREAD

**Discussion by nozzlegear** [crbednarz/moonkinmetrics#4] [Ideas]
Hey! Loving your website. One thing that I'd like to see is the distribution of stats – i.e. haste, mastery, crit, etc. – for each spec. It'd be really interesting if those stats could be tied to the different talent builds as well. Though once it gets narrowed down enough I'm sure the stats might be more affected by the individual player's vault luck than any cohesive build they're trying to create.

**Comment by crbednarz** [comment]
This is a really interesting idea, and I think it fits well into what the site does. Although I do wonder how much variance there is for stats within a spec. 

From a technical perspective, those stats are small enough where they'd likely they could still fit into the page without extra API requests.

The real hurdle is that it'll double the amount of scanning against the wow API to get that data. This is something that's already likely to occur for other enhancements, but has quite a bit more work before it's ready.

(Apparently I don't get email notifications for new discussions- sorry for the delay)
